libflux/flux-core/src/semantic/tests/vectorize.rs

```diff
@@ -1,24 +1,33 @@
 use super::*;
 use crate::semantic::{
-    import::Packages,
+    bootstrap,
     nodes::{FunctionExpr, Package},
     walk::{walk, Node},
     AnalyzerConfig, Feature,
 };
+use std::path::PathBuf;
 
 fn analyzer_config() -> AnalyzerConfig {
     AnalyzerConfig {
         features: vec![
             Feature::VectorizedMap,
             Feature::VectorizedConditionals,
             Feature::VectorizedEqualityOps,
+            Feature::VectorizedFloat,
         ],
         ..AnalyzerConfig::default()
     }
 }
 
 fn vectorize(src: &str) -> anyhow::Result<Package> {
-    let mut analyzer = Analyzer::new(Default::default(), Packages::default(), analyzer_config());
+    let stdlib_path = PathBuf::from(concat!(env!("CARGO_MANIFEST_DIR"), "/../../stdlib"))
+        .canonicalize()
+        .expect("stdlib path");
+
+    let cfg = analyzer_config();
+    let (prelude, imports, _sem_pkgs) = bootstrap::infer_stdlib_dir(&stdlib_path, cfg.clone())?;
+
+    let mut analyzer = Analyzer::new((&prelude).into(), &imports, cfg);
```

Comment:
Is this inferring the whole stdlib every time we receive a request?

Reply:
I think "yes" but more specifically, note that this is happening for regular scripts regardless.

This particular analyzer is used for these tests specific tests - the change here has no impact on the regular execution of flux otherwise.

Previously there was no need for the prelude (or any other part of the stdlib) to be visible during these test runs. Probably this was skipped 
- to be faster
- it was needless (before now)
- (my observation working on this issue) it creates a lot of noise if you're watching in a debugger

Reply:
Oh, and to speak to why this is needed -- without this (or something similar) vectorization will fail in these tests because the program will error immediately with `undefined identifier: float`.

Reply:
Ah, ok. I missed that this was test code.